docs/pages/demo.rst

```diff
@@ -72,6 +72,11 @@ You should end up with the following directory structure::
         ...
     demo_app.py                                     # The thermostat node script
 
+``CYPHAL_PATH`` should contain a list to all the paths where the DSDL root namespace directories are to be found:
+
+..  code-block:: sh
+    export CYPHAL_PATH="$HOME/pycyphal-demo/custom_data_types:$HOME/pycyphal-demo/public_regulated_data_types"
```

Comment:
The docs build failed because of the missing newline here.

```suggestion

    export CYPHAL_PATH="$HOME/pycyphal-demo/custom_data_types:$HOME/pycyphal-demo/public_regulated_data_types"
```